.gemini/settings.json

```diff
@@ -2,12 +2,12 @@
 {
     "mcpServers": {
         "github": {
-            "command": "/server/github-mcp-server",
+            "command": "dev/mcp/github-mcp-server",
             "args": [
                 "stdio"
             ],
             "env": {
-                "GITHUB_READ-ONLY": "1"
+                "GITHUB_READ_ONLY": "1"
```

Comment:
Good fix on the environment variable name. `GITHUB_READ_ONLY` is the correct convention and likely what the server expects.